packages/storybook/src/stories/3-ConcertoForm.stories.js

```diff
@@ -55,6 +60,42 @@ export const Demo = () => {
     }
   };
 
+  options.relationshipProvider = {
+    getOptions : (field) => {
+      if(field.getFullyQualifiedTypeName() === 'test.Person') {
```

Comment:
Why is this necessary? Is field `null` on an early render, perhaps?

Reply:
No, I just wanted to illustrate how to define a relationship provider that is sensitive to the type of the relationship.